vhdl/hdlworkflow.m

```diff
@@ -3,9 +3,18 @@
 dataplane_name = 'dataplane';
 load_system(model);
 
+if mp.target_system == 'de10'
+    device_family = 'Cyclone V';
+    device = '5CSEBA6U23I7';
+elseif mp.target_system == 'arria10'
+    device_family = 'Arria 10';
+    device = '10AS066H2F34I1HG';
+end
```

Comment:
Do you know if the HDL workflow advisor requires the device and device family to be set? It would seem reasonable that we shouldn't need that information for the 'Generic ASIC/FPGA' workflow, but for some reason I think we might need them. If you don't know off the top of your head, this is worth looking into, as we could simplify the HDL workflow script if these parameters aren't needed.

Reply:
I don't although I wouldn't be surprised if it doesn't require it. I will look into it